src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -1175,6 +1209,7 @@ function addCORSHeaders(res: PostGraphileResponse): void {
       'Content-Length',
       // For our 'Explain' feature
       'X-PostGraphile-Explain',
+      'X-PostGraphile-Explain-Options',
```

Comment:
Thinking about it, I'm very keen that we don't introduce another header because it means people may have to rewrite their nginx, CORS and similar config.